src/components/TripList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import TripListItem from './TripListItem'
+import * as Trips from '../data/trips.json'
+
+export default function TripList() {
+  const tripData = Trips.trips
```

Comment:
Trips.trips sounds a little bit redundent imho, maybe find a better name in the `trips.json`?